src/components/Manifesto.tsx

```diff
@@ -2,111 +2,85 @@ import React from "react";
 import styled from "styled-components";
 import { useTranslation } from "react-i18next";
 import {
-  color,
   layout,
   space,
   typography,
-  background,
   flexbox,
-  border,
   grid,
-  ColorProps,
   LayoutProps,
   SpaceProps,
   TypographyProps,
-  BackgroundProps,
   FlexboxProps,
-  BorderProps,
   GridProps,
 } from "styled-system";
+import Header from "./Header";
 
-const Main = styled.main<
-  ColorProps & SpaceProps & BackgroundProps & LayoutProps & FlexboxProps
->`
+const Main = styled.main<SpaceProps & FlexboxProps>`
   min-height: 100vh;
-  overflow: hidden;
-  ${color};
+  display: flex;
   ${space};
-  ${background};
-  ${layout};
   ${flexbox}
 `;
 
-const H1 = styled.h1<ColorProps & TypographyProps & SpaceProps>`
-  text-transform: uppercase;
-  ${color};
-  ${typography};
+const Container = styled.div<SpaceProps & LayoutProps>`
   ${space};
+  ${layout};
 `;
 
-const Grid = styled.div<GridProps & FlexboxProps & SpaceProps>`
+const Grid = styled.div<GridProps & FlexboxProps>`
   display: grid;
   ${grid};
   ${flexbox};
-  ${space};
 `;
 
-const Container = styled.div<
-  ColorProps & SpaceProps & BorderProps & LayoutProps
->`
-  ${color};
+const H1 = styled.h1<TypographyProps & SpaceProps & GridProps>`
+  text-transform: uppercase;
+  ${typography};
   ${space};
-  ${border}
-  ${layout};
+  ${grid};
 `;
 
-const Paragraph = styled.p<ColorProps & TypographyProps & SpaceProps>`
+const Paragraph = styled.p<TypographyProps & SpaceProps>`
   text-transform: uppercase;
-  ${color};
   ${typography};
   ${space};
+  padding-bottom: 20px;
+  text-align: justify;
 `;
 
 const Manifesto = () => {
   const { t } = useTranslation();
+  const fontSizes = [1, 2, null, null, 2, 3, 4];
 
   return (
-    <Main
-      bg="black"
-      px={[1, 4, 6]}
-      pt={5}
-      display="flex"
-      alignItems="center"
-      justifyContent="center"
-    >
-      <Container p={4} display="block">
-        <H1 color="athensGray" fontSize={[2, 4, 5]} pb={5}>
+    <Main p={6} alignItems="center" justifyContent="center">
+      <Header />
```

Comment:
ive just commented on one of the issues for adding the Header to all the pages. Its best if its done in one place in the parent of all the routes

Reply:
let's tackle this on a separate PR 👍